test/kongintegration/containers/kong.go

```diff
@@ -47,13 +50,21 @@ func NewKong(ctx context.Context, t *testing.T) Kong {
 	})
 	require.NoError(t, err)
 
+	kong := Kong{
+		container: kongC,
+	}
+	adminURL, err := url.Parse(kong.AdminURL(ctx, t))
+	require.NoError(t, err)
+
+	kongVersion, err := helpers.ValidateMinimalSupportedKongVersion(adminURL, consts.KongTestPassword) //nolint:contextcheck
```

Comment:
Do you mind propagating the context through the call stack to not introduce the `nolint`?

Reply:
Done (it was quite a lot to do 😛)